superset/tags/commands/create.py

```diff
@@ -96,13 +94,10 @@ def run(self) -> None:
     def validate(self) -> None:
         exceptions = []
         # Validate object_id
-        if self._objects_to_tag:
-            if any(obj_id == 0 for obj_type, obj_id in self._objects_to_tag):
-                exceptions.append(TagInvalidError())
-
+        if objects_to_tag := self._properties.get("objects_to_tag", []):
```

Comment:
same here.. is this if statement necessary if objects_to_tag is an empty array, then the for loop before will do nothing.